src/domain/space/account/account.resolver.mutations.ts

```diff
@@ -106,6 +106,12 @@ export class AccountResolverMutations {
         community: true,
       },
     });
+    if (!rootSpace) {
```

Comment:
If we're supporting Accounts without spaceId, do we need to throw an error here?

Reply:
yes, we have just created a space a few lines ago, this should return the space for sure, but now that `getRootSpace` was nullable typescript was complaining. 
But this code is gone in the Neil's PR. I'm going to merge it for now.